packages/kbn-mock-idp-plugin/public/login_page.tsx

```diff
@@ -105,6 +108,39 @@ export const LoginPage = () => {
                     placeholder="Enter your name"
                     css={{ width: 350 }}
                   />
+                  <Field
+                    as={EuiInlineEditTitle}
+                    name="username"
+                    heading="h5"
+                    type={config.uiam?.enabled ? 'number' : 'text'}
+                    inputAriaLabel="Edit username inline"
+                    value={formik.values.username}
+                    onChange={(event: ChangeEvent<HTMLInputElement>) => {
+                      formik.setFieldValue(
+                        'username',
+                        config.uiam?.enabled
+                          ? event.target.value
+                          : sanitizeUsername(event.target.value)
+                      );
+                    }}
```

Comment:
I tested the new field for the current "role" login and `username` works as expected.